.prettierignore

```diff
@@ -0,0 +1,2 @@
+*.md
```

Comment:
Prettier does help a lot in auto-formatting markdown files and re-organizing / creating links at the top of READMEs, etc. Are you sure we want to exclude it? If the only issue is related to the release notes, can you change the glob pattern to only ignore release notes md files?

Reply:
Only major issue seems to be release-notes. Other thing was that I knew we had some documentation that was based of Opensearch-Dashboards and since there were issues with release-notes I assumed other issues would exist. Doesn't seem like it after further checking so I'll change it to only ignore release notes md files.

Reply:
Gotcha, yeah that makes sense. I think we're ok to just ignore release notes for now, since that is the only md file that will cause workflows to break if formatting is off.